libs/common/src/helpers/date.test.ts

```diff
@@ -82,7 +82,7 @@ describe('date helpers', () => {
       const result = getISO8601();
       const year = now().getFullYear();
       const month = (now().getMonth() + 1).toString().padStart(2, '0');
-      const date = now().getDate();
+      const date = now().getDate().toString().padStart(2, '0');
```

Comment:
This is fine and not the main point of this PR so no need to change - but just to raise that generally to test functions like this it is best to use `jest.useFakeTimers` so that the date cannot change and the tests being run on different days are consistent. One can then also use different describe blocks to test different dates - e.g. one with a one digit date and one with a 2 digit date to get better coverage.

Reply:
Can be done later, should have a tech debt ticket @abhi-markan